lib/executors/Executor.js

```diff
@@ -297,10 +297,6 @@ define([
 				})
 				.finally(lang.bind(self, '_afterRun'))
 				.then(function () {
-					if (self._hasSuiteErrors) {
-						throw new Error('One or more suite errors occurred during testing');
```

Comment:
The overall test run should fail if there is a suite error which is what this was doing. I don’t think it really matters how it comes out at the end as long as it’s a non-zero exit code for the run.